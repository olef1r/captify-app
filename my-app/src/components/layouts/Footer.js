import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Typography from '@material-ui/core/Typography';
import Link from '@material-ui/core/Link';
import Container from '@material-ui/core/Container';

function Copyright() {
  return (
    <Typography variant="body2" color="textSecondary" align="center">
      {'Copyright Â© '}
      <Link color="inherit" href="https://material-ui.com/">
        Your Website
      </Link>{""}
        {new Date().getFullYear()}
        {'.'}
    </Typography>
  );
}

const useStyles = makeStyles(theme => ({
  toolbar: {
    borderBottom: `1px solid ${theme.palette.divider}`,
  },

  
  overlay: {
    position: 'absolute',
    top: 0,
    bottom: 0,
    right: 0,
    left: 0,
    backgroundColor: 'rgba(0,0,0,.3)',
  },
  mainFeaturedPostContent: {
    position: 'relative',
    padding: theme.spacing(3),
    [theme.breakpoints.up('md')]: {
      padding: theme.spacing(6),
      paddingRight: 0,
    },
  },
  footer: {
    backgroundColor: theme.palette.background.paper,
    marginTop: theme.spacing(8),
    padding: theme.spacing(6, 0),
  },
}));

export default function Blog() {
  const classes = useStyles();

  return (
    <React.Fragment>
      <footer className={classes.footer}>
        <Container maxWidth="lg">
          <Typography variant="h6" align="center" gutterBottom>
            Footer
          </Typography>

          <Copyright />
        </Container>
      </footer>
      </React.Fragment>
  );
}