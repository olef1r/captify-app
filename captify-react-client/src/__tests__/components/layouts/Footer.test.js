import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../../components/layouts/Footer';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Footer/>);
});

describe('<Footer> - Component', () => {
  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
    expect(wrapper).toIncludeText('Footer')
  });
});