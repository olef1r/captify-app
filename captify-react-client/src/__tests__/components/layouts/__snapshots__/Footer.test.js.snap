// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> - Component renders without crashing 1`] = `
<Fragment>
  <footer
    className="makeStyles-footer-4"
  >
    <WithStyles(ForwardRef(Container))
      maxWidth="lg"
    >
      <WithStyles(ForwardRef(Typography))
        align="center"
        gutterBottom={true}
        variant="h6"
      >
        Footer
      </WithStyles(ForwardRef(Typography))>
      <Copyright />
    </WithStyles(ForwardRef(Container))>
  </footer>
</Fragment>
`;
